### PR TITLE
Column reordering and selection

### PR DESCRIPTION
**Pull request by guillermo-carrasco**
Add column drag and drop reordering and column show/hide button.

screen shot 2016-02-17 at 15 24 45

### CODE REVIEW

File: puzzle/server/blueprints/variants/templates/variants.html [hunk 1]
```diff
@@ -65,9 +65,12 @@ <h4 class="panel-title">
     $('.selectpicker').selectpicker();
     $(document).ready(function() {
       $('#variants_list').DataTable({
+        dom: 'Bfrtip',
```
Comment (robinandeer) [inline puzzle/server/blueprints/variants/templates/variants.html]: what does this do?

**Comment by robinandeer** [comment]
Not a big deal but we're starting to get so many scripts and CSS files - it would be nice to bundle them all!

I also think that Puzzle should work offline which means we need to serve resources locally

For a different PR

**Comment by robinandeer** [comment]
Very nice additions, I approve!!

File: puzzle/server/blueprints/variants/templates/variants.html (hunk 1 above)
Comment (guillermo-carrasco) [inline puzzle/server/blueprints/variants/templates/variants.html]: https://www.datatables.net/extensions/buttons/ look at "dom parameter" :-)

**Comment by guillermo-carrasco** [comment]
Totally agree about the CSS/JS loading. No so sure about the point of working offline, since Puzzle links to lots of external sources, doesn't it?

But yeah, bundle bundleee!

**Pull request merged**